x/ethbridge/keeper/msg_server_test.go

```diff
@@ -0,0 +1,92 @@
+package keeper_test
```

Comment:
Can you also add test to assert behaviour when we pass `SetPauser` with a non-admin account?